src/main/java/seedu/address/ui/StatisticsDisplay.java

```diff
@@ -44,12 +42,12 @@ public StatisticsDisplay() {
         super(FXML);
         this.barChartTitleText = DEFAULT_BAR_CHART_TITLE;
         this.data = null; // TODO *******************
-        this.progressBarDailyFilepath = DEFAULT_PROGRESS_BAR_DAILY__PLACEHOLDER;
+        this.progressBarDailyFilepathString = DEFAULT_PROGRESS_BAR_DAILY_PLACEHOLDER;
         this.progressDailyText = DEFAULT_PROGRESS_DAILY;
         this.medalsText = DEFAULT_MEDALS;
 
         progressDaily.setText(progressDailyText);
-        Image progressBarDailyImage = new Image(String.valueOf(progressBarDailyFilepath));
+        Image progressBarDailyImage = new Image(DEFAULT_PROGRESS_BAR_DAILY_PLACEHOLDER);
```

Comment:
Recommend to use this, was used in duke and ab. Online also recommended this.
The slash in the filepath is fine, duke/ab also uses the slash
this.getClass().getResourceAsStream(DEFAULT_PROGRESS_BAR_DAILY_PLACEHOLDER));